torch/_inductor/codegen/wrapper.py

```diff
@@ -473,10 +475,15 @@ def generate_end(self, result):
         return
 
     def generate_extern_kernel_alloc(self, extern_kernel, args):
+        ending = self.ending
+        if config.memory_planning and "view_as_complex" in str(extern_kernel.kernel):
+            # view operation fallbacks cause issues since inductor
+            # doesn't know the memory is still needed and might reuse it.
+            ending = f".clone(){ending}"
```

Comment:
cc @htyu 

we will need to remove this workaround once we have a lowering for `view_as_complex`

Reply:
We model the aliasing and memory reuse on `ComplexView` node (https://github.com/pytorch/pytorch/blob/c14c4efc0eb97ccc471ac6cbdcc659417227ae97/torch/_inductor/ir.py#L4446) . Is that sufficient?

Reply:
Yeah that should work, just need to handle aten.view_as_complex.